src/internal/pfsdb/commit_provenance.go

```diff
@@ -240,16 +192,17 @@ func GetSubvenantCommits(ctx context.Context, ext sqlx.ExtContext, startId Commi
 	if err != nil {
 		return nil, errors.Wrap(err, "get commit with id subvenance")
 	}
+	return rowToCommits(ctx, ext, subvenantCommits)
+}
+
+func rowToCommits(ctx context.Context, ext sqlx.ExtContext, rows []*CommitRow) ([]*Commit, error) {
 	var commits []*Commit
-	for _, commit := range subvenantCommits {
-		commitInfo, err := getCommitInfoFromCommitRow(ctx, ext, commit)
+	for _, row := range rows {
+		commit, err := row.ToCommit(ctx, ext)
 		if err != nil {
-			return nil, errors.Wrap(err, "get commit with id subvenance")
+			return nil, errors.Wrap(err, "get commit with id provenance")
```

Comment:
[nit] May I ask why the error msg is "...id provenance"? rowToCommits is also used with subvenantCommits.

Reply:
this probably got refactored into a smaller function at one point and I forgot to update the error messages.